sunpy/net/dataretriever/tests/test_norh.py

```diff
@@ -83,28 +187,49 @@ def test_wavelength_range():
 def test_query_wrong_wave():
     c = norh.NoRHClient()
     with pytest.raises(ValueError):
-        c.search(a.Time("2016/10/1", "2016/10/2"), a.Instrument('norh'),
-                 a.Wavelength(50*u.GHz))
+        c.search(a.Time("2016/10/1", "2016/10/2"), a.Instrument('norh'), a.Wavelength(50*u.GHz))
 
 
-@pytest.mark.remote_data
-@pytest.mark.parametrize("time,instrument,wave", [
-    (a.Time('2012/10/4', '2012/10/6'), a.Instrument('norh'), a.Wavelength(17*u.GHz)),
-    (a.Time('2013/10/5', '2013/10/7'), a.Instrument('norh'), a.Wavelength(34*u.GHz))])
-def test_get(time, instrument, wave):
+@pytest.mark.usefixtures('create_mock_search')
```

Comment:
```suggestion
@pytest.mark.usefixtures('create_mock_search', 'create_mock_url')
```

Reply:
Also I suggest we put this decorator on the line above the function definition.

Reply:
Done